src/tools/sfdk/sdkmanager.cpp

```diff
@@ -1352,6 +1352,10 @@ bool SdkManager::mapEnginePaths(QString *program, QStringList *arguments, QStrin
         qCDebug(sfdk) << "Mapping" << mapping.hostPath << "as" << mapping.enginePath;
         program->replace(mapping.hostPath, mapping.enginePath, mapping.cs);
         arguments->replaceInStrings(mapping.hostPath, mapping.enginePath, mapping.cs);
+        if (QDir::toNativeSeparators(mapping.hostPath) != mapping.hostPath) {
+            program->replace(QDir::toNativeSeparators(mapping.hostPath), mapping.enginePath, mapping.cs);
+            arguments->replaceInStrings(QDir::toNativeSeparators(mapping.hostPath), mapping.enginePath, mapping.cs);
+        }
```

Comment:
What if we have e.g. `C:\Users\user\foo\bar` in arguments and `C:\Users\user` is the shared home path. Wouldn't it result to `/home/mersdk/share\foo\bar`?

Reply:
Indeed it did. Fixed now.